packages/cloud_firestore/cloud_firestore_platform_interface/lib/src/settings.dart

```diff
@@ -103,9 +143,49 @@ class Settings {
         host,
         sslEnabled,
         cacheSizeBytes,
+        webExperimentalForceLongPolling,
+        webExperimentalAutoDetectLongPolling,
+        webExperimentalLongPollingOptions,
         ignoreUndefinedProperties,
       );
 
   @override
   String toString() => 'Settings($asMap)';
 }
+
+/// Options that configure the SDK’s underlying network transport (WebChannel) when long-polling is used
+/// These options are only used if experimentalForceLongPolling is true
+/// or if experimentalAutoDetectLongPolling is true and the auto-detection determined that long-polling was needed.
+/// Otherwise, these options have no effect.
```

Comment:
This one can be shorten to match ```Options that configure the SDK’s underlying network transport (WebChannel) when long-polling is used.``` on https://firebase.google.com/docs/reference/js/firestore_.experimentallongpollingoptions.md#experimentallongpollingoptions_interface